frontend/src/theme/components/Tile.ts

```diff
@@ -0,0 +1,67 @@
+import { ComponentMultiStyleConfig } from '@chakra-ui/theme'
+
+const activeContainerStyle = {
+  bgColor: 'primary.200',
+  borderColor: 'transparent',
+  // NOTE: For the boxShadow styling, due to conflicts with the focus-visible package,
+  // the !important is required to display the boxShadow styling correctly.
+  boxShadow: '0 0 0 3px var(--chakra-colors-primary-500) !important',
+}
+
+const baseContainerStyle = {
+  color: 'inherit',
+  borderRadius: '0.25rem',
+  padding: '1.5rem',
+  _active: activeContainerStyle,
+  _hover: {
+    bgColor: 'primary.100',
+  },
+  _focus: {
+    borderColor: 'transparent',
+    // NOTE: For the boxShadow styling, due to conflicts with the focus-visible package,
+    // the !important is required to display the boxShadow styling correctly.
+    boxShadow: '0 0 0 2px var(--chakra-colors-primary-500) !important',
+  },
+  bgColor: 'white',
+  border: '1px solid',
+  borderColor: 'neutral.300',
+  whiteSpace: 'pre-line',
+  flexDir: 'column',
+  alignItems: 'flex-start',
+  maxWidth: 'inherit',
+  textAlign: 'left',
+}
+
+export const Tile: ComponentMultiStyleConfig = {
+  parts: ['container', 'title', 'icon', 'subtitle'],
+  baseStyle: ({ isActive }) => ({
+    container: isActive
+      ? { ...baseContainerStyle, ...activeContainerStyle }
```

Comment:
nit: do we really need two constants for this? should we inline them instead for something that is easier to read?

i.e.

```ts
container: {
  bg: isActive? 'primary.200' : 'white',
  ...
}
```

This allows the reader to know exactly what the differences between active and non-active states are without having to scroll up and down between the two constants and see what is being replaced

Reply:
i did this because the activeState is shared between `_active` and when `isActive` is true! i think this is worth the overhead but lmk if this is impt to change!